app/keepers/keepers.go

```diff
@@ -277,26 +272,19 @@ func NewAppKeeper(
 		govRouter,
 	)
 
+	// TODO: Confirm TransferKeeper wiring correct after strangelove backports https://github.com/strangelove-ventures/packet-forward-middleware/blob/6867bcaac00fef1267c4ea99c3cbb6ee8bc6a025/router/keeper/keeper.go#L49
 	appKeepers.TransferKeeper = ibctransferkeeper.NewKeeper(
 		appCodec,
 		appKeepers.keys[ibctransfertypes.StoreKey],
 		appKeepers.GetSubspace(ibctransfertypes.ModuleName),
-		appKeepers.IBCKeeper.ChannelKeeper,
+		appKeepers.RouterKeeper,
```

Comment:
This looks like its not going to work as expected because the `RouterKeeper` is initialised below on line 314. 
At this point it looks like we're going to be passing an empty struct.

Reply:
yeah, TransferKeeper need routerKeeper, routerKeeper need transferKeeper to initialize, one must be empty

Reply:
@damiannolan Strangelove have added [a setter on `RouterKeeper` for the `TransferKeeper`.](https://github.com/strangelove-ventures/packet-forward-middleware/commit/4fb516ae522e1a36376ca856b82d9d6e75284213)